plenum/test/pp_seq_no_restoration/test_node_erases_stored_last_sent_pp_key_on_pool_restart.py

```diff
@@ -0,0 +1,108 @@
+from plenum.common.constants import LAST_SENT_PRE_PREPARE
+from plenum.test import waits
+from plenum.test.checkpoints.conftest import chkFreqPatched
+from plenum.test.helper import sdk_send_batches_of_random, assertExp
+from plenum.test.pool_transactions.helper import \
+    disconnect_node_and_ensure_disconnected
+from plenum.test.test_node import ensureElectionsDone, getPrimaryReplica, \
+    checkNodesConnected
+from plenum.test.view_change.helper import start_stopped_node
+from stp_core.loop.eventually import eventually
+
+CHK_FREQ = 2
+LOG_SIZE = 3 * CHK_FREQ
+
+nodeCount = 4
+
+backup_inst_id = 1
+
+
+def test_node_erases_stored_last_sent_pp_key_on_pool_restart(
```

Comment:
This tests stops a tested node, then restarts other nodes in the pool, then start the tested node.
It would be great to write another test, where all nodes (including the tested one) are restarted at the same time